app/experimenter/experiments/tasks.py

```diff
@@ -123,8 +125,61 @@ def update_experiment_bug_task(user_id, experiment_id):
 @app.task
 @metrics.timer_decorator("update_experiment_info.timing")
 def update_experiment_info():
-    metrics.incr("update_experiment_info.started")
-    logger.info("Updating experiment info")
+    update_ready_to_ship_experiments()
+    update_launched_experiments()
```

Comment:
Don't you have to call `.delay()` here to actually dispatch it as a separate task?